packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/UnbrandedTypeSpecClient.cs

```diff
@@ -21,6 +21,9 @@ public UnbrandedTypeSpecClient()
             _pipeline = ClientPipeline.Create();
         }
 
+        /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
+        public ClientPipeline Pipeline => _pipeline;
```

Comment:
do we need the field if this is a public property?

Reply:
Removed the field.